app/Http/Controllers/ForwardChatworkController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Webhook;
+use App\Models\Condition;
+use Illuminate\Http\Request;
+
+class ForwardChatworkController extends Controller
+{
+    public function forwardMessage(Request $params, $token)
+    {
+        $webhook = Webhook::where('token', '=', $token)->first();
+        $payloads = $webhook->payloads;
+
+        $conditions = Condition::whereIn('payload_id', $payloads->pluck('id'))->get();
+
+        $condition = $conditions->first()->condition;
+
+        foreach($conditions as $condition) {
+            if (eval($condition->condition)) {
+                $payload = $condition->payload;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Http/Controllers/ForwardChatworkController.php</br>**Line**: 24</br>Blank line found at end of control structure